src/test/java/com/google/sps/NonPersistentUserRepositoryTest.java

```diff
@@ -102,7 +103,9 @@ public void multipleAddAndRemove() {
       //myRepo should have 2 users stored
       Assert.assertEquals(2, allUsers.size());
 
-      myRepo.removeUser(userB);
+      try {
+        myRepo.removeUser(userB);
+      } catch (Exception e) {}
```

Comment:
Maybe add an Assert.fail inside the catch block? That way the test would fail if your program threw an exception unexpectedly for (validly) removing an existing user.

Reply:
Done, thanks for the suggestion!